src/api/http-client.ts

```diff
@@ -0,0 +1,65 @@
+export class HttpClient {
+  public async makeRequest(request: Request): Promise<any> {
```

Comment:
Consider using `unknown` instead of `any`:

> Much like `any`, any value is assignable to `unknown`; however, unlike `any`, you cannot access any properties on values with the type `unknown`, nor can you call/construct them. Furthermore, values of type `unknown` can only be assigned to `unknown` or `any`.

> ...we’re now forced to either perform checking, or use a type assertion to convince the type-system that we know better.

source: [The `unknown` type](https://blogs.msdn.microsoft.com/typescript/2018/07/12/announcing-typescript-3-0-rc/#the-unknown-type)

```suggestion
  public async makeRequest(request: Request): Promise<unknown> {
```

Reply:
Consider changing all the methods of `HttpClient` to be `static` methods.

```suggestion
  public static async makeRequest(request: Request): Promise<any> {
```

Reply:
thanks @rmjordas will make these changes.